### ISSUE TITLE
On traversing through tree items in Left Pane under 'View Inspect' tab item, keyboard focus is improper

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#390]
### Steps to Reproduce

1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Create a Workbook with WPF flatform.
3. Run the sample workbook code.
4. Invoke "View Inspector" tab and navigate through the tree elements present in left pane using keyboard.

### Expected Behavior
Should be able to navigate through tree elements in the left pane without any issues
### Actual Behavior
After setting focus on tree elements in the left pane if user presses down arrow, focus goes to next level (below) element on the tree. If user again tries navigate down through the tree by using down arrow by pressing down arrow focus is going to Menu bar (File Tab). User has to travel again till the tree to go for next element. This repeats for all the tree elements in the tree for both upward/downward navigation over the tree elements.
### User Impact:
Keyboard user who rely on keyboard will be confused with order of navigation. As every time user has to traverse through whole menu to get on to design tree, lot of time will be consumed for user.
### Attachment:
Please find the video taken with Inspect focus
[Video.zip](https://github.com/Microsoft/workbooks/files/2057170/Video.zip)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
